src/dotnet/App.Maui/App.Maui.csproj

```diff
@@ -458,4 +467,20 @@
     <Message Importance="High" Text="Service Worker has been copied." />
   </Target>
 
+  <!-- Target needed until LinkWithSwiftSystemLibraries makes it into the SDK: https://github.com/xamarin/xamarin-macios/pull/20463 -->
```

Comment:
maybe mark with TODO:?

Reply:
I think it will not work when this target will be released as part of xamarin workloads